eth/main.cpp

```diff
@@ -773,9 +773,12 @@ int main(int argc, char** argv)
     if (vm.count("help"))
     {
         cout
-                << "Usage eth [OPTIONS]\n"
-                << "Options:\n\n"
-                << "Wallet usage:\n";
+			<< "NAME:\n"
+			<< "   eth - the cpp-ethereum command line interface\n\n"
+			<< "   Copyright 2013-2018 The cpp-ethereum Authors\n\n"
```

Comment:
This would be the second place where we'd need to keep the year up-to-date. Could we somehow not duplicate it here and somehow reuse the string returned by `credits()` from `MinerAux.h` ?